rust/src/delta.rs

```diff
@@ -1577,4 +1624,48 @@ mod tests {
             assert!(parquet_filename.contains("col1=a/col2=b/part-00000-"));
         }
     }
+
+    #[tokio::test]
+    async fn test_create_delta_table() {
+
+        let test_schema = Schema::new(
+            "test".to_string(),
+            vec![
+                SchemaField::new(
+                    "Id".to_string(), 
+                    SchemaDataType::primitive("integer".to_string()),
+                    true,
+                    HashMap::new()
+                ),
+                SchemaField::new(
+                    "Name".to_string(), 
+                    SchemaDataType::primitive("string".to_string()),
+                    true,
+                    HashMap::new()
+                )
+            ]
+        );
+
+        let deltamd = DeltaTableMetaData::new(
+            None, 
+            None, 
+            test_schema, 
+            vec![], 
+            HashMap::new()
+        );
```

Comment:
It might be really helpful to test what would be the generated value of `DeltaTableMetadata` attributes: `id`, `created_time`, `format` 😄

Reply:
ill add some asserts, thanks. How would you mock out file creation, so that when test run, files are just created somewhere? Or should a test include the write action, but just remove the files at the end?

Reply:
You can perform the write in a temp dir, then remove the directory at the end of the test. For example: https://github.com/delta-io/delta-rs/blob/a7328d6644688c79d618285aa4bd4ab728159340/rust/src/storage/file/mod.rs#L132